.github/workflows/airtable.yaml

```diff
@@ -23,9 +29,17 @@ jobs:
           chmod +x airtable-auto.py
           ./airtable-auto.py
           
-      - name: Create Pull Request
-        uses: colbymorrison/create-pull-request@v2
-        with:
-          commit-message: "auto-add new news"
-          title: "Airtable GitHub action -- Update News"
-          body: "New News from Airtable"
+      - name: Update master and prod
+        run: |
+          git fetch --all
+
+          git add assets/data/medialist.json
+          git commit -m "auto update news from airtable"
+
+          git checkout master
+          # Only update medialist.json 
+          git checkout auto-add-news assets/data/medialist.json
+          git commit -m "auto update news from airtable"
```

Comment:
I think, based on [this SO thread](https://stackoverflow.com/questions/3878624/how-do-i-programmatically-determine-if-there-are-uncommitted-changes), that you can run:

```bash
git diff-index --cached --quiet HEAD || git commit -m "auto update news from airtable"
```

and avoid committing if there are no changes.

Reply:
Nice! From a quick test with a local repo that seems to work.